.github/workflows/build-vcpkg.yml

```diff
@@ -117,6 +117,35 @@ jobs:
       asset-name: 'docker-ubuntu-22_04'
     secrets: inherit
 
+  check-documentation-changes:
+    if: ${{ contains('pull_request,push', github.event_name) }}
+    runs-on: ubuntu-22.04
+    outputs:
+      documentation_contents_changed: ${{ steps.variables.outputs.documentation_contents_changed }}
+    steps:
+      - name: Check for Documentation Changes
+        id: changed
+        uses: dorny/paths-filter@v3
+        with:
+          filters: |
+            src:
+              - 'docs/**'
```

Comment:
You could add `.github/workflows/test-documentation.yml` to this?

Reply:
reusable workflows have to be called from individual jobs (apparently). I tried adding it as a step but it would fail. Documentation on the subject always seems to reference reusable workflows as standalone jobs.

Reply:
No I meant add that file as a path to monitor? - IOW if you have a future PR which changes this file, it will trigger a docs build on its PR...

Reply:
ah. Yeah let me make sure that works.

Reply:
Added it to the file path. Will run if we update that workflow now.